dashboard/plugins/table_descriptions/__init__.py

```diff
@@ -7,13 +7,15 @@
 tab_name = 'Table descriptions'
 plugin = Blueprint('table_descriptions', __name__, template_folder='templates')
 
-def get_descriptions(cache_ttl=300):
+DEFAULT_CACHE = 300
```

Comment:
So we have default defined in `settings.py` and here? In case a user leaves cache option empty in settings?

Reply:
Indeed :-)